<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flow_AccumulationDialogBase</class>
 <widget class="QDialog" name="Flow_AccumulationDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flow Accumulation</string>
  </property>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>25</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Flow Direction GRID :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Output Grid :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLayers">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>20</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOutput">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>60</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenDialog">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkEdge">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>90</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Edge Contamination</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
